### DISCUSSION TITLE
Get annotation text

### DISCUSSION THREAD

**Discussion by ghost** [GateNLP/python-gatenlp#204] [Help Q&A] [answered]
Hi,

I have a corpus of annotated documents in *.gate format, I load the document and iterate over the annotations in a specific annotation set but can't get the text, instead I get information about the annotation:

```
doc = Document.load('myDoc.gate', fmt='gatexml')
annSet = doc.annset('myAnnSet')
for ann in annSet.with_type('typeX'):
    print(ann)
```

which gives me:

```
Annotation(975,984,typeX,features=Features({}),id=83999)
Annotation(1427,1449,typeX,features=Features({'feature1': '17', 'feature2': 'ABCD'}),id=83286)
```

Is it possible to get the actual text (as not all annotation have features and I just need to text)?

Thanks!

**Comment by johann-petrak** [comment, marked as answer]
Yes you can use the annotation as a slice like that:  `doc[ann]`

**Reply by ghost** [reply]
Thanks Johann!
